java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -68,7 +71,24 @@ abstract class AbstractArcHost(
     constructor(
         schedulerProvider: SchedulerProvider,
         vararg initialParticles: ParticleRegistration
-    ) : this(Dispatchers.Default, schedulerProvider, null, *initialParticles)
+    ) : this(Dispatchers.Default, Dispatchers.Default, schedulerProvider, null, *initialParticles)
+
+    /**
+     * Backward-compatible for some existing [AbstractArcHost] implementations to dispatch
+     * [contextSerializationTasks] onto the [coroutineContext] instead of separate context.
+     */
+    constructor(
```

Comment:
Can we mark this as `@Deprecated`, including a `replaceWith` param?

Reply:
sure! lemme add it